specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/securityStandards.json

```diff
@@ -0,0 +1,391 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2024-08-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/securityStandards": {
+      "get": {
+        "x-ms-examples": {
+          "List security standards by subscription scope": {
+            "$ref": "./examples/SecurityStandards/ListBySubscriptionSecurityStandards_example.json"
+          },
+          "List security standards by security connector scope": {
+            "$ref": "./examples/SecurityStandards/ListBySecurityConnectorSecurityStandards_example.json"
+          },
+          "List security standards by management group scope": {
+            "$ref": "./examples/SecurityStandards/ListByManagementGroupSecurityStandards_example.json"
+          }
+        },
+        "tags": [
+          "SecurityStandards"
+        ],
+        "description": "Get a list of all relevant security standards over a scope",
+        "operationId": "SecurityStandards_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
```

Comment:
>OK [](http://example.com/codeflow?start=28&length=2)

please fix all descriptions , wherever possible , to be more descriptive

Reply:
Fixed.